src/App.tsx

```diff
@@ -1,10 +1,43 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Status } from './types/Status';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+const USER_ID = 11092;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [todosFilterBy, setTodosFilterBy] = useState<Status>(Status.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch((error) => {
+        setIsError(true);
+        throw error;
+      });
+  }, []);
+
+  const preparedTodos = useMemo(() => {
+    return todos.filter(todo => {
+      switch (todosFilterBy) {
+        case Status.ACTIVE:
+          return !todo.completed;
+        case Status.COMPLETED:
+          return todo.completed;
+        default:
+          return true;
+      }
+    });
+  }, [todosFilterBy, todos]);
```

Comment:
Not fixed
![2023-08-07_16h28_50](https://github.com/mate-academy/react_todo-app-loading-todos/assets/61904995/b950e7d7-1684-4ae1-9079-a8b67f46fd30)